killer_subscriber.go

```diff
@@ -0,0 +1,36 @@
+package signal
+
+import (
+	"log"
+	"os"
+	"syscall"
+)
+
+var signalsKiller = []os.Signal{os.Kill, os.Interrupt, syscall.SIGHUP, syscall.SIGTERM, syscall.SIGQUIT, syscall.SIGSTOP}
+var killerSubscriber func()
```

Comment:
I don't get why do you need to have a global var here

Reply:
this subscriber must be instanciate only once

Reply:
ok :)

Reply:
can you add a comment on that line ?